app/code/Magento/Catalog/Pricing/Price/ConfiguredRegularPrice.php

```diff
@@ -57,23 +57,22 @@ public function __construct(
      * @param ItemInterface $item
      * @return $this
      */
-    public function setItem(ItemInterface $item) : ConfiguredRegularPrice
+    public function setItem(ItemInterface $item)
```

Comment:
Do not remove existing type hints, feel free to add new ones.

Reply:
Thanks for your review. I revert it back.